pyopenms_viz/_core.py

```diff
@@ -796,6 +811,13 @@ def plot(self):
             self.canvas, ann_texts, ann_xs, ann_ys, ann_colors
         )
 
+        # Add peptide sequence if configured
+        if hasattr(self._config, "display_peptide_sequence") and self._config.display_peptide_sequence:
+            peptide_sequence = kwargs.get("peptide_sequence", None)
+            matched_fragments = kwargs.get("matched_fragments", [])
+            if peptide_sequence:
+                self.add_peptide_sequence(peptide_sequence, matched_fragments)
```

Comment:
These variables should be stored automatically in the object. Thus you should be able to reference them using self.peptide_sequence and self.matched_fragments. This will remove the need for kwargs